libandroid-navigation-ui/src/test/java/com/mapbox/services/android/navigation/ui/v5/instruction/InstructionLoaderTest.java

```diff
@@ -0,0 +1,99 @@
+package com.mapbox.services.android.navigation.ui.v5.instruction;
+
+import android.widget.TextView;
+
+import com.mapbox.api.directions.v5.models.BannerComponents;
+import com.mapbox.api.directions.v5.models.BannerText;
+
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+public class InstructionLoaderTest {
+
+  @Test
+  public void testPriorityInfoIsAdded() {
```

Comment:
@Guardiola31337 I feel like that style doesn't really apply here because I'm testing functionality as a whole, not just a method. Also the method names would sound weird without `test`. It would be `priorityInfoIsAdded`, sounds like something is missing. When sprinting weren't we using names like this? @danesfeder what are your thoughts as well

Reply:
I agree here that `test` can be omitted due to the `@Test` annotation and `InstructionLoaderTest` class name already implying that it is a test.  

I also agree with the naming convention @Guardiola31337 is suggesting here - something like `onInstructionLoaderCreated_priorityInfoIsAdded`.  Looking at the test code, that's what I understood was being tested.

During the sprint, we didn't go into much detail about naming, but fwiw, when looking at the wayname PR, we were following this same type of structure https://github.com/mapbox/mapbox-navigation-android/pull/853/files#diff-31e5dc0b3d24264f2e6b5daeff6d0710R20